Configurations/unix-Makefile.tmpl

```diff
@@ -828,22 +828,22 @@ errors:
 CRYPTOHEADERS={- join(" \\\n\t", sort @cryptoheaders) -}
 SSLHEADERS={- join(" \\\n\t", sort @sslheaders) -}
 ordinals:
-	( cd $(SRCDIR); \
+	( cd "$(SRCDIR)"; \
           $(PERL) util/mknum.pl --version $(VERSION) --no-warnings \
                   --ordinals util/libcrypto.num \
                   --symhacks include/openssl/symhacks.h \
                   $(CRYPTOHEADERS) )
-	( cd $(SRCDIR); \
+	( cd "$(SRCDIR)"; \
           $(PERL) util/mknum.pl --version $(VERSION) --no-warnings \
                   --ordinals util/libssl.num \
                   --symhacks include/openssl/symhacks.h \
                   $(SSLHEADERS))
 
 test_ordinals:
 	( cd test; \
-	  SRCTOP=../$(SRCDIR) \
-	  BLDTOP=../$(BLDDIR) \
-	    $(PERL) ../$(SRCDIR)/test/run_tests.pl test_ordinals )
+	  SRCTOP="../$(SRCDIR)" \
+	  BLDTOP="../$(BLDDIR)" \
+	    $(PERL) "../$(SRCDIR)/test/run_tests.pl" test_ordinals )
```

Comment:
BLDDIR is somthing like "."
and this PR is not adding support for SRCDIR with spaces, right?

Reply:
As I said in the beginning my goal was to fix the case of DSTDIR having spaces. Then I applied the same class of fix to SRCDIR. There is also a caveat here (noted in one the commit messages), that SRCDIR (and also BLDDIR in 1.1.1x) is used in GNU Make rule definitions, where spaces need more work to be supported (quoting doesn't work). So I'd be guessing that SRCDIR/BLDDIR having spaces still wouldn't work at this point.

Reply:
Okay, I see, sorry for confusion.
maybe better leave SRCDIR for another PR.

Reply:
After spending a good amount of time on it, I'd be happier to include this now. It's a step into the right direction, even if not complete at this point.

Reply:
Okay, for me.